src/main/java/org/terasology/hunger/HungerAuthoritySystem.java

```diff
@@ -114,16 +110,20 @@ public void postBegin(){
      * Deals a unit of hunger damage to the character.
      */
     @ReceiveEvent
-    public void onPeriodicActionTriggered(PeriodicActionTriggeredEvent event, EntityRef entity_unused) {
+    public void onPeriodicActionTriggered(PeriodicActionTriggeredEvent event, EntityRef entityUnused) {
         if (event.getActionId().equals(HUNGER_DAMAGE_ACTION_ID)) {
             for (EntityRef entity : entityManager.getEntitiesWith(HungerComponent.class, AliveCharacterComponent.class)) {
                 HungerComponent hunger = entity.getComponent(HungerComponent.class);
+                hunger.lastCalculatedFood = Math.max(0,
```

Comment:
This is not _necessary_, as the [`HungerWindow.java`](https://github.com/Terasology/Hunger/blob/master/src/main/java/org/terasology/hunger/ui/HungerWindow.java#L51) uses [method](https://github.com/Terasology/Hunger/blob/master/src/main/java/org/terasology/hunger/HungerUtils.java#L33) based on time calculation. 
I don't know which is better in terms of performance.